stock_picking_analytic/models/stock_picking.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    original_analytic_account_id = fields.Many2one(
+        comodel_name="account.analytic.account",
+        help="Technical field. Use to store the value of"
+        "analytic_account_id if there is no lines",
+    )
+    analytic_account_id = fields.Many2one(
+        compute="_compute_analytic_account_id",
+        inverse="_inverse_analytic_account_id",
+        comodel_name="account.analytic.account",
+        string="Analytic Account",
+        readonly=True,
+        states={
+            "draft": [("readonly", False)],
+            "waiting": [("readonly", False)],
+            "confirmed": [("readonly", False)],
+            "assigned": [("readonly", False)],
+        },
+        store=True,
+    )
+
+    @api.depends(
+        "move_ids_without_package.analytic_account_id", "original_analytic_account_id"
```

Comment:
@rousseldenis Was there a reason to use `move_ids_without_package` here instead of `move_lines` (same with all the other places where `move_ids_without_package` is used in this module)?

I am proposing to use `move_ids` (the field renamed from `move_lines`) in the 16.0 migration https://github.com/OCA/account-analytic/pull/527#issue-1551809876 and I want to know if I was overlooking something.

Reply:
@yostashiro I don't know (I cherry-picked the work of my colleague :sweat_smile: ). He said to me it was something about the view (but no more details)